src/fal/cli/flow_runner.py

```diff
@@ -95,3 +108,55 @@ def _unique_id_to_model_name(unique_id: str):
 
 def _unique_ids_to_model_names(id_list: List[str]):
     return list(map(lambda id: _unique_id_to_model_name(id), id_list))
+
+
+def _combine_fal_run_results(target_path):
+    last_run_result_file = os.path.join(target_path, "run_results.json")
+    last_run_result = _get_all_result_content(last_run_result_file)
+    last_run_result_array: List[str] = last_run_result["results"]
```

Comment:
```suggestion
    last_run_result_array = _get_result_array(os.path.join(target_path, "run_results.json"))
```

Reply:
I need `last_run_result` below in the same function